### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception com.intellij.ide.ui.UITheme$1@309300bf cannot patch icon path java.lang.StringIndexOutOfBoundsException: String index out of range: 0

### ISSUE THREAD

**Issue by jschuwan** [microsoft/azure-tools-for-java#6795]
IntelliJ build version: 2021.3.3 IU-213.7172.25
OS: Mac OS X
JDK: JetBrains s.r.o. 11.0.14.1
Plugin version: 3.66.1-2021.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,0,1520x1135,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=com.intellij.util.ui.UIUtil$1@49bc1d9a,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=com.intellij.ide.ui.UITheme$1@309300bf cannot patch icon path, throwable=java.lang.StringIndexOutOfBoundsException: String index out of range: 0
	at java.base/java.lang.StringLatin1.charAt(StringLatin1.java:47)
	at com.microsoft.azure.toolkit.intellij.common.IntelliJAzureIcons.loadIcon(IntelliJAzureIcons.java:109)
	at com.microsoft.azure.toolkit.intellij.common.IntelliJAzureIcons.lambda$doGetIcon$2(IntelliJAzureIcons.java:91)
	at com.microsoft.azure.toolkit.intellij.common.IntelliJAzureIcons.doGetIcon(IntelliJAzureIcons.java:91)
	at com.microsoft.azure.toolkit.intellij.common.IntelliJAzureIcons.getIcon(IntelliJAzureIcons.java:82)
	at com.microsoft.azure.toolkit.intellij.common.action.IntellijAzureActionManager$AnActionWrapper.applyView(IntellijAzureActionManager.java:145)
	at com.mi

**Comment by Flanker32** [comment]
@jschuwan Thanks for your report, we have resolved this issue in latest toolkit v3.67.0, could you please help upgrade to the latest toolkit and try again?

**Comment by Flanker32** [comment]
close as duplicated with #6758
